src/main/java/com/regalii/regaliator/api/Request.java

```diff
@@ -86,8 +85,7 @@ public Response delete(final String endpoint) {
 
     private Response requestWithBody(final String httpMethod, final String endpoint, final Map<String, Object> params) {
         final String json = JSON.dump(params);
-        final String md5 = MD5.digest(json);
-        final HttpURLConnection connection = buildConnection(httpMethod, endpoint, md5);
+        final HttpURLConnection connection = buildConnection(httpMethod, endpoint, null);
```

Comment:
If we no longer send the MD5, will be better to remove it from the method signature for `buildConnection`, instead of send `null`